ui/components/TopMenu/TopMenuProtocolList.tsx

```diff
@@ -48,49 +53,139 @@ export default function TopMenuProtocolList({
   onProtocolChange,
 }: TopMenuProtocolListProps): ReactElement {
   const { t } = useTranslation()
+  const history = useHistory()
   const currentNetwork = useBackgroundSelector(selectCurrentNetwork)
   const showTestNetworks = useBackgroundSelector(selectShowTestNetworks)
   const productionNetworks = useBackgroundSelector(selectProductionEVMNetworks)
 
+  const customNetworksEnabled = isEnabled(FeatureFlags.SUPPORT_CUSTOM_NETWORKS)
+
   return (
     <div className="standard_width_padded center_horizontal">
-      <ul>
-        {productionNetworks.map((network) => (
-          <TopMenuProtocolListItem
-            isSelected={sameNetwork(currentNetwork, network)}
-            key={network.name}
-            network={network}
-            info={
-              productionNetworkInfo[network.chainID] ||
-              t("protocol.compatibleChain")
-            }
-            onSelect={onProtocolChange}
-            isDisabled={disabledChainIDs.includes(network.chainID)}
-          />
-        ))}
-        {showTestNetworks && testNetworks.length > 0 && (
-          <>
-            <li className="protocol_divider">
-              <div className="divider_label">
-                {t("topMenu.protocolList.testnetsSectionTitle")}
-              </div>
-              <div className="divider_line" />
-            </li>
-            {testNetworks.map((info) => (
-              <TopMenuProtocolListItem
-                isSelected={sameNetwork(currentNetwork, info.network)}
-                key={info.network.name}
-                network={info.network}
-                info={info.info}
-                onSelect={onProtocolChange}
-                isDisabled={info.isDisabled ?? false}
-              />
-            ))}
-          </>
-        )}
-      </ul>
+      <div className={classNames(customNetworksEnabled && "networks_list")}>
+        <ul>
```

Comment:
Isn't classnames syntax like:
```suggestion
      <div className={classNames({ "networks_list": customNetworksEnabled })}>
        <ul>
```

Reply:
Yes, both ways work since it compacts the parameters array by removing falsy values